doc/Porting.md

```diff
@@ -251,4 +251,16 @@ of a board that is merged into the main Tock repository:
    and help test the platform for future releases.
 
 With these requirements met we should be able to merge the platform into Tock
-relatively quickly.
+relatively quickly. In the pull request to add the platform, you should add this
+checklist:
+
+```md
+### New Platform Checklist
+
+- [ ] Hardware is widely available.
```

Comment:
And a link to acquire the hardware is available in the board README (?)

Reply:
We could certainly go into a lot more detail about what we expect: links, rust doc headers, no unwrap, etc., but maybe that gets a little clunky for a PR checklist?